app/Controllers/indexController.php

```diff
@@ -266,10 +266,9 @@ public static function listEntriesByContext(?int $postsPerPage = null): Traversa
 		}
 
 		foreach ($entryDAO->listWhere(
-					$type, $id, FreshRSS_Context::$state, FreshRSS_Context::$order,
-					$postsPerPage ?? FreshRSS_Context::$number, FreshRSS_Context::$offset, FreshRSS_Context::$first_id,
-					FreshRSS_Context::$search, $date_min)
-				as $entry) {
+			$type, $id, FreshRSS_Context::$state, FreshRSS_Context::$order,
+			$postsPerPage ?? FreshRSS_Context::$number, FreshRSS_Context::$offset, FreshRSS_Context::$first_id,
+			FreshRSS_Context::$search, $date_min) as $entry) {
```

Comment:
Also clarity reduction.

Reply:
Disabling `Squiz.ControlStructures.ForEachLoopDeclaration.SpacingBeforeAs` completely would prevent checking spaces in `foreach ($list as $item) {}`, and since the problem occurs only once in the whole codebase, I came with another workaround

Reply:
Oh, I didn't mean to insist too strongly. :-)